app/views/users/_user_list.html.erb

```diff
@@ -14,7 +14,7 @@
                       <p><%= presenter.title_list %></p>
                   <% end %>
                   <% if user.country_name %>
-                      <p><%= presenter.country_name %></p>
+                      <p title="<%= presenter.country_name unless presenter.country_name.length < 15 %>"><%= truncate(presenter.country_name, length:14) %></p>
```

Comment:
this feels like too much logic in the view - either pull to a helper, or check if you even need the unless statement